_episodes/03-configuration.md

```diff
@@ -3,91 +3,236 @@ title: "Configuration"
 teaching: 0
 exercises: 0
 questions:
-- "What is user configuration file and how can I use it?"
+- "What is the user configuration file and how should I use it?"
 objectives:
-- "Understand the data directories structure"
-- "Configure ESMValTool to ignore some settings"
+- "Understand the contents of the user-config.yml file"
+- "Prepare a personalized user-config.yml file"
+- "Configure ESMValTool to use some settings"
 keypoints:
-- "The ``config-user.yml`` file tells ESMValTool what data are input"
-- "The ``config-user.yml`` file tells ESMValTool what directory is the destination"
+- "The ``config-user.yml`` tells ESMValTool where to find input data."
+- " ``rootpath`` defines the root directory for the input data."
+- " ``output_dir`` defines the destination directory."
 ---
 
 ## The configuration file
 
-The ``config-user.yml`` configuration file contains all the global level information needed by ESMValTool to run.
-This is an (YAML file) [https://yaml.org/spec/1.2/spec.html]. An example configuration file can be found in the root directory of the ESMValTool repository.
-Make a copy and rename it to ``config-user.yml``:
-
-~~~
-  cp config-user-example.yml config-user.yml
+The ``config-user.yml`` configuration file contains all the global level information
+needed by ESMValTool to run. This is an
+[YAML file](https://yaml.org/spec/1.2/spec.html). An example configuration file
+can be found in the root directory of the ESMValTool repository:
+[config-user-example.yml](https://github.com/ESMValGroup/ESMValTool/blob/master/config-user-example.yml).
+
+Let's download it to our working directory ``esmvaltool_tutorial``
+that was created during the [Setup]({{ page.root }}{% link setup.md %}).
```

Comment:
The setup.md is of #46 and the installation instructions of #31 , don't create an `esmvaltool_tutorial` directory. Can the directory reason and creation be described here or in  #31 or in #46?

Reply:
now the directory creation is explained inside this lesson.